specification/confidentialledger/resource-manager/readme.typescript.md

```diff
@@ -6,8 +6,8 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ```yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "confidentialledger"
-  output-folder: "$(typescript-sdks-folder)/packages/confidentialledger"
+  package-name: "mgmt-confidentialledger"
```

Comment:
please use @azure/arm-confidentialledger as the package name. the @azure/arm- prefix is for all mgmt RP.